app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -252,12 +252,12 @@
 // modal overflow scrolling
 .onboarding-modal-scroll-container {
   height: auto;
-  min-height: 436px;
+  max-height: calc(100vh - 289px);
```

Comment:
I got the 289px from tweaking around in a few browser. If there's a better way to go about this please let me know.

Reply:
This looks good for now. As we roll out this type of modal experience to other surfaces, we'll track and see if there are more optimal approaches.